src/projects/automation/workflows/trigger.md

```diff
@@ -0,0 +1,19 @@
+# Trigger
+
+A trigger node allows you to run other workflow as part of the current workflow.
+
+The trigger node works in tandem with a :docs-link[Listener]{id="projects/automation/workflows/listener"} to start a workflow by making an API call to the listener, which in turn starts the workflow. This allows you to design your workflows in a more modular way, which encourages better design and maintainability.
+
+![Trigger Diagram](/src/assets/trigger_diagram.png){width="60%"}
```

Comment:
this format doesn't work. use `:img{src="/src/assets/rigger_diagram.png" alt="Trigger Diagram" width="60%" }`